docs/source/howto/index.md

```diff
@@ -0,0 +1,19 @@
+(howto)=
+
+# How-To Guides
+
+The documentation in this section is _goal-oriented_ and designed to guide the
+reader through specific steps to reach that goal.
+
+```{note}
+[Read more about the How-To quadrant of the diataxis framework.](https://diataxis.fr/how-to-guides/)
+```
+
+```{attention}
+This documentation is a Work in Progress!
+Please see our [contributing guide](contributing) if you'd like to add to it.
+```
+
+```{toctree}
+install.md
```

Comment:
Non-blocking comment, but just my opinion.

I actually think installation guides are the one case that breaks the "How Do I..." title for How-To guides and should be a tutorial. My reasoning is this:

- Tutorials end up with everyone who completes them having the _exact same_ result (e.g. produced the same graph) because it's about learning how to do something specific (use a feature)
- How-to guides end up with everyone who completes them having _roughly similar_ results (e.g. the result is "authentication is enabled" but some folk may have used GitHub and others CILogon) because it's about achieving a goal (enabling authentication) but there may be different pathways to the goal (this auth provider over that one and the steps may be slightly different)
- Installation instructions are therefore tutorials because everyone who completes them ends up with the _same result_ (e.g. the environment set up correctly, the software running, the thing correctly installed)

My other reason is that howto guides are traditionally mentioned second after tutorials and this IMO puts installation instructions too far down for the user experience.

Reply:
Ah, these are extremely valid points. Will come back on this tomorrow. Thank you @sgibson91 ❤️